nixos/modules/hardware/graphics.nix

```diff
@@ -101,7 +101,7 @@ in
     assertions = [
       {
         assertion = cfg.enable32Bit -> pkgs.stdenv.hostPlatform.isx86_64;
-        message = "`hardware.graphics.enable32Bit` only makes sense on a 64-bit system.";
+        message = "`hardware.graphics.enable32Bit` only makes sense on a x86_64 system.";
```

Comment:
"an", since it's "an ex-eighty-six". Also, any ideas on how we could reword this whole thing? "Only makes sense" doesn't make sense when you don't already know why it doesn't make sense.

Reply:
How about " `hardware.graphics.enable32Bit` is only supported on an x86_64 system currently. " ?

Reply:
Something like that, yeah. Maybe drop the "currently" even.

Reply:
> Something like that, yeah. Maybe drop the "currently" even.

Is there any chance that we may support to run 32bit applications on risv64 or aarch64 platform(?

I have asked some expert friends, it's hardly to see on non x86 platforms but just want to confirm once.

Reply:
Most hardware out there doesn't really support that, we can revisit if there's ever an actual need.